src/annotator/test/guest-test.js

```diff
@@ -189,18 +195,21 @@ describe('Guest', () => {
 
   describe('events from host frame', () => {
     describe('on "sidebarLayoutChanged" event', () => {
-      it('calls fitSideBySide if `Guest` is the main annotatable frame', () => {
```

Comment:
Are these two removed tests no longer meaningful? Is "main annotatable frame" implicitly the case if the guest frame is filling the host frame?

Reply:
> Is "main annotatable frame" implicitly the case if the guest frame is filling the host frame?

In all current situations, yes. It is possible in future there may be a situation where the main content is in an iframe that doesn't fill the host frame. We'll probably have to change how side-by-side mode works in that scenario.